Resources/Prototypes/Loadouts/Jobs/Heads/headOfPersonnel.yml

```diff
@@ -0,0 +1,42 @@
+- type: loadout
+  id: LoadoutCommandHOPNeckMantle
+  name: HOP Mantle
+  description: To show who has the authority around here.
+  category: Jobs
+  cost: 2
+  exclusive: true
+  jobWhitelist:
+    - HeadOfPersonnel
+  item: ClothingNeckMantleHOP
+
+- type: loadout
+  id: LoadoutcommandHOPNeckCloak
+  name: HOP Cloak
+  description: To really show who has the authority around here.
+  category: Jobs
+  cost: 2
+  exclusive: true
+  jobWhitelist:
+    - HeadOfPersonnel
+  item: ClothingNeckCloakHop
+
+- type: loadout
+  id: LoadoutcommandHOPBackIan
+  name: Ian Backpack
+  description: A backpack that looks like Ian, how cute!
+  category: Jobs
+  cost: 4
+  exclusive: true
```

Comment:
As the person who wrote some of the code for this, I am concerned this will _delete every item already in the user's backpack._ This should be non-exclusive, otherwise I think special care will need to be taken for backpack-items.

Reply:
Oh, whoops